dotcom-rendering/src/web/components/ImageComponent.tsx

```diff
@@ -273,7 +276,6 @@ export const ImageComponent = ({
 					isLazy={!isMainMedia}
 					isMainMedia={isMainMedia}
 				/>
-				{starRating && <PositionStarRating rating={starRating} />}
```

Comment:
Note: removed review stars from immersive layout. @oliverlloyd has a PR to add them back into the correct place

Reply:
Do you have the link to the PR?

Reply:
I actually ended up putting this work aside. It's not complicated but we don't have a use case for showing stars on immersive pieces at this point so there's no urgency to this